lib/rubocop/cop/layout/empty_line_between_defs.rb

```diff
@@ -157,14 +159,18 @@ def module_candidate?(node)
           cop_config['EmptyLineBetweenModuleDefs'] && node.module_type?
         end
 
-        def message(node)
-          type = case node.type
-                 when :def, :defs
-                   :method
-                 else
-                   node.type
-                 end
-          format(MSG, type: type)
+        def message(node, count: nil)
+          type = node_type(node)
+
+          if allowance_range?
```

Comment:
I would be great if for default config `NumberOfEmptyLines=1` and multiple lines, the message mentions that `%<n>d lines were detected, 1 expected`.

Reply:
Thanks for the suggestion @tejasbubane! Now the message shows the actual and expected count even when it's not a range.